cmd/statusd/debug/debug.go

```diff
@@ -0,0 +1,104 @@
+package debug
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"net"
+	"reflect"
+
+	"github.com/status-im/status-go/geth/api"
+)
+
+// Debug provides a server receiving line based commands from a
+// CLI via the debugging port and executing those on the Status API
+// using reflection. The returned values will be rendered as
+// string and returned to the CLI.
+type Debug struct {
+	csv      reflect.Value
+	listener net.Listener
+}
+
+// New creates a debugger using the oassed Status API.
+// It also starts the server.
+func New(statusAPI *api.StatusAPI) (*Debug, error) {
+	listener, err := net.Listen("tcp", ":51515") // nolint
```

Comment:
`nolint` is still required?

Reply:
Got a *Binds to all network interfaces* by our linter. How to care for if all interfaces are okay?

Reply:
Interesting. I didn't know we have a check for that. It's ok then. I guess it will disappear when we switch to support both IP address + port.